protocol-units/mock-assets/devnet/m1/mevm/src/interfaces/IWETH10.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (C) 2015, 2016, 2017 Dapphub
+// Adapted by Ethereum Community 2021
+pragma solidity >=0.7.6;
+
+import "./IERC20.sol";
+import "./IERC2612.sol";
+import "./IERC3156FlashLender.sol";
+
+/// @dev Wrapped Ether v10 (WETH10) is an Ether (ETH) ERC-20 wrapper. You can `deposit` ETH and obtain a WETH10 balance which can then be operated as an ERC-20 token. You can
+/// `withdraw` ETH from WETH10, which will then burn WETH10 token in your wallet. The amount of WETH10 token in any wallet is always identical to the
+/// balance of ETH deposited minus the ETH withdrawn with that specific wallet.
+interface IWETH10 is IERC20, IERC2612, IERC3156FlashLender {
```

Comment:
IWETH10 back? Let's stick with 9

Reply:
I mean, maybe good to test it. It doesn't conflict with anything and I'd say it's an interesting statement.

Reply:
This is probably out of scope for this PR.  Let's leave this as 9.